src/app/src/util/util.js

```diff
@@ -20,17 +22,12 @@ import {
 } from './constants';
 
 export function DownloadCSV(data, fileName) {
-    const csvData = new Blob([data], { type: 'text/csv;charset=utf-8;' });
-    if (window.navigator.msSaveOrOpenBlob) {
-        // IE hack; see http://msdn.microsoft.com/en-us/library/ie/hh779016.aspx
-        window.navigator.msSaveBlob(csvData, fileName);
-    } else {
-        const csvURL = window.URL.createObjectURL(csvData);
-        const tempLink = document.createElement('a');
-        tempLink.href = csvURL;
-        tempLink.setAttribute('download', fileName);
-        tempLink.click();
-    }
+    saveAs(
+        new Blob([data], { type: 'text/csv;charset=utf-8;' }),
+        fileName,
```

Comment:
In testing, when downloading from Chrome, the file had a `.csv` extension, but neither Firefox nor Safari did. Maybe we have to explicitly add it here?

Reply:
Interesting! Looks like we do. I left this function as is -- so callers will have to supply the extension -- but updated it here  77f2948fb0e727f7e66ab700477d27feca33bbf6